lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2321,6 +2324,18 @@ SubExprInfo ExprEmitter::visitSV(GetModportOp op) {
   return {Selection, IsUnsigned};
 }
 
+SubExprInfo ExprEmitter::visitSV(SystemFunctionOp op) {
+  if (hasSVAttributes(op))
+    emitError(op, "SV attributes emission is unimplemented for the op");
+
+  ps << "$" << op.getFnName() << "(";
+  llvm::interleave(
+      op.getOperands(), [&](Value v) { emitSubExpr(v, LowestPrecedence); },
+      [&]() { ps << PP::nbsp << "," << PP::space; });
+  ps << ")";
+  return {Symbol, IsUnsigned};
```

Comment:
I think some of these might be signed, like `$signed`, maybe this is part of the expression op (attribute)?